File: api/src/main/java/net/blueshell/api/service/CommitteeMemberService.java
package net.blueshell.api.service;

import net.blueshell.api.base.BaseModelService;
import net.blueshell.api.model.committee.CommitteeMember;
import net.blueshell.api.repository.committee.CommitteeMemberRepository;
import org.springframework.stereotype.Service;

@Service
public class CommitteeMemberService extends BaseModelService<CommitteeMember, CommitteeMemberRepository> {

    public CommitteeMemberService(CommitteeMemberRepository repository) {
        super(repository);
    }
}

File: api/src/main/kotlin/net/blueshell/api/service/CommitteeMemberService.kt
package net.blueshell.api.service

import net.blueshell.api.service.base.BaseModelService
import net.blueshell.api.model.committee.CommitteeMember
import net.blueshell.api.model.committee.CommitteeMemberId
import net.blueshell.api.repository.committee.CommitteeMemberRepository
import org.springframework.stereotype.Service

@Service
class CommitteeMemberService(repository: CommitteeMemberRepository) :
    BaseModelService<CommitteeMember, CommitteeMemberId, CommitteeMemberRepository>(repository)
